manifests/cluster/deployment.yaml

```diff
@@ -13429,6 +13429,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - get
+  - list
+
```

Comment:
These permissions must be defined by the kubebuilder and generated by the kustomize.

Reply:
I think canfeng has already added permission kubebuilder annotations and this is the regenerating result.

Reply:
the generated files are ignored by git

cc @heww 

@wangcanfengxs  please confirm it.

Reply:
`manifests/cluster/deployment.yaml` should update.
`config/rbac/role.yaml` will regenerate, but ignored by git